src/molecule_docker/playbooks/create.yml

```diff
@@ -28,19 +28,19 @@
 
     - name: Check presence of custom Dockerfiles
       ansible.builtin.stat:
-        path: "{{ molecule_scenario_directory + '/' + (item.dockerfile | default( 'Dockerfile.j2')) }}"
+        path: "{{ molecule_scenario_directory + '/' + (item.dockerfile | default('Dockerfile.j2')) }}"
       loop: "{{ molecule_yml.platforms }}"
       register: dockerfile_stats
 
     - name: Create Dockerfiles from image names
       ansible.builtin.template:
         # when using embedded playbooks the dockerfile is alonside them
         src: >-
-          {%- if dockerfile_stats.results[i].stat.exists -%}
```

Comment:
Are you really sure that removing the "-" does not break the template? AFAIK the linter does not touch these and altering them has high risk of damaging the generate template.

Reply:
I see. Sorry to mark it as a draft PR.
It's lucky we have test to stop me doing this.

Currently, there is an extra space between before `else`.

Actual: `}} {%- else -%}` 
Expected by linter: `}}{%- else -%}`

Reply:
It seems only if we put all statement in one single line, the linter can pass.
`{%- if foo -%}...{%- endif -%}`

But it will make the line too long.

@ssbarnea 
Do you have some suggestion?
https://github.com/ansible-community/molecule-docker/runs/8033866754?check_suite_focus=true#step:9:122

Reply:
Do you still have any concern about this PR?

`Using double quote and back slash` is the only way supporting folding without spaces.

> Using a “Folded Block Scalar” > will fold newlines to spaces;

But ansible-lint is not happy with the `spaces`.

Reference: 
https://docs.ansible.com/ansible/2.9/reference_appendices/YAMLSyntax.html
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210

Reply:
@ssbarnea 

Sorry to bother you.

Could you kindly review this PR?